### DISCUSSION TITLE
Does the change in Elasticsearch's/Kibana's license mean anything to Security Onion?

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#2641] [Unsupported Versions] [answered]
I do not know much about licensing, but I've read some articles about the SSPL license that's going to be used now, and not a lot of people seemed to be too happy about the license.

From what I understood, the parts where most complaints were about shouldn't influence Security Onion, as it already is completely open source and is not offered as SaaS, but I might have missed something.

So the title says it all:
Does the change in Elasticsearch's/Kibana's license mean anything to Security Onion?

**Comment by dougburks** [comment, marked as answer]
https://blog.securityonion.net/2021/01/upcoming-elastic-licensing-changes.html
